invenio_groups/forms.py

```diff
@@ -19,26 +19,21 @@
 
 """Group Forms."""
 
-from __future__ import absolute_import, print_function, unicode_literals
-
-from invenio_base.i18n import _
-from invenio.utils.forms import InvenioBaseForm
+from __future__ import absolute_import, print_function
 
+from flask_babelex import gettext as _
 from sqlalchemy_utils.types.choice import ChoiceType
-
-from wtforms import RadioField, TextAreaField
+from wtforms import Form, RadioField, TextAreaField
```

Comment:
I think we should depend on ``from flask_wtf import Form`` since it provides CSRF protection out-of-the-box.